log_outgoing_requests/management/commands/cleanup_requests_logs.py

```diff
@@ -0,0 +1,8 @@
+from django.core.management.base import BaseCommand
+
+from log_outgoing_requests.models import OutgoingRequestsLog
```

Comment:
within a django app, use relative imports

```suggestion
from ...models import OutgoingRequestsLog
```